Easydict/NewApp/View/MenuView/MainMenuShortcutCommand.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  MainMenuShortcutCommand.swift
+//  Easydict
+//
+//  Created by Sharker on 2024/2/4.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import SwiftUI
+
+extension EasyDictMainMenu {
+    struct MainMenuShortcutCommand: Commands {
+        @State private var appShortcutCommandList = [
+            MainMenuShortcutCommandDataItem(title: "shortcut_clear_input", type: .clearInput),
+            MainMenuShortcutCommandDataItem(title: "shortcut_clear_all", type: .clearAll),
+            MainMenuShortcutCommandDataItem(title: "shortcut_copy", type: .copy),
+            MainMenuShortcutCommandDataItem(title: "shortcut_copy_first_teanslated_text", type: .copyFirstResult),
+            MainMenuShortcutCommandDataItem(title: "shortcut_focus", type: .focus),
+            MainMenuShortcutCommandDataItem(title: "shortcut_play", type: .play),
+            MainMenuShortcutCommandDataItem(title: "retry", type: .retry),
+            MainMenuShortcutCommandDataItem(title: "toggle_languages", type: .toggle),
+            MainMenuShortcutCommandDataItem(title: "pin", type: .pin),
+            MainMenuShortcutCommandDataItem(title: "hide", type: .hide),
+            MainMenuShortcutCommandDataItem(title: "shortcut_increase_font", type: .increaseFontSize),
+            MainMenuShortcutCommandDataItem(title: "shortcut_decrease_font", type: .decreaseFontSize),
+            MainMenuShortcutCommandDataItem(title: "open_in_google", type: .google),
+            MainMenuShortcutCommandDataItem(title: "open_in_eudic", type: .eudic),
+            MainMenuShortcutCommandDataItem(title: "open_in_apple_dictionary", type: .appleDic),
+        ]
+
+        var body: some Commands {
+            // shortcut Commands
+            CommandMenu("shortcut") {
+                ForEach(appShortcutCommandList) { item in
+                    MainMenuShortcutCommandItem(dataItem: item)
+                    if item.title == "toggle_languages" || item.title == "shortcut_decrease_font" {
```

Comment:
<img width="735" alt="Screenshot 2024-02-07 at 00 01 43" src="https://github.com/tisfeng/Easydict/assets/103433299/7e024303-b99c-473d-a7a2-1bfec6b84db1">


I think this logic could be moved into `MainMenuShortcutCommandItem` component.